lib/metrics/processHandles.js

```diff
@@ -14,13 +17,24 @@ module.exports = (registry, config = {}) => {
 
 	const gauge = new Gauge({
 		name: namePrefix + NODEJS_ACTIVE_HANDLES,
-		help: 'Number of active handles.',
+		help: 'Number of active handles grouped by handle type.',
+		labelNames: ['type'],
+		registers: registry ? [registry] : undefined
+	});
+	const totalGauge = new Gauge({
```

Comment:
We don't need a total and by type, you can just not select by labels

Reply:
@SimenB I kept the separate total metric to keep backward compatibility.
If I will drop then guys who rely on this metric can loose their alerts and their graphics will be broken. I think its better to avoid breaking changes if possible.

Reply:
since we did not have labels before, I don't think it's a breaking change to add them. unless people group by labels, it'll look the ezxact same

Reply:
@SimenB metric names are different. 
Original "total" gauge has name "nodejs_active_requests_total" and "nodejs_active_handles_total"
New gauges has name "nodejs_active_requests" and "nodejs_active_handles"

So everybody has to explicitly change prometheus queries from nodejs_active_requests_total to sum(nodejs_active_requests).

Reply:
Yes, my point is that we _don't_ need new metrics since these should just be labels, right?

Reply:
@SimenB  if I use same name (nodejs_active_requests_total and nodejs_active_handles_total)
then everyone have to change query from nodejs_active_handles_total to sum(nodejs_active_handles_total). And this is breaking change.

My point was to avoid make breaking changes.

Reply:
Right, I agree this seems best then. Thanks!